sale_invoice_policy/models/sale_order.py

```diff
@@ -1,43 +1,22 @@
 # Copyright 2017 ACSONE SA/NV (<http://acsone.eu>)
+# Copyright 2021 Digital5, S.L. (<https://digital5.es>)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo import api, fields, models
+from odoo import fields, models
 
 
 class SaleOrder(models.Model):
 
     _inherit = "sale.order"
 
     invoice_policy = fields.Selection(
-        [("order", "Ordered quantities"), ("delivery", "Delivered quantities")],
+        [
+            ("order", "Invoice what is ordered"),
+            ("delivery", "Invoice what is delivered"),
+        ],
+        string="Invoicing Policy",
         readonly=True,
         states={"draft": [("readonly", False)], "sent": [("readonly", False)]},
-        help="Ordered Quantity: Invoice based on the quantity the customer "
-        "ordered.\n"
-        "Delivered Quantity: Invoiced based on the quantity the vendor "
-        "delivered (time or deliveries).",
+        help="Ordered Quantity: Invoice quantities ordered by the customer.\n"
+        "Delivered Quantity: Invoice quantities delivered to the customer.",
     )
-    invoice_policy_required = fields.Boolean(
```

Comment:
This module changes the standard behaviour. It's better to keep it optional.

Reply:
I don't understand what you are talking about: are you meaning the deletion of the `invoice_policy_required` field is not "standard"?
As I said the option to make required `invoice_policy` on sale.order makes this module not compatible with others like `sale_automatic_workflow`.

Reply:
It seems the comment from @rousseldenis has made this PR review process not active, while asked question not yet replied